src/utils.jl

```diff
@@ -0,0 +1,61 @@
+using CompilerPluginTools, YaoLocations
+
+"""
+    random_circuit(nbits, ngates [, two_qubits_ratio, t_ratio])
+
+Generate a random quantum circuit of qubit number `nbits` and with gate number `ngates`,
+where 2-qubit gate number is of ratio `two_qubits_ratio` and T-gate number of ratio `t_ratio`.
+"""
+function random_circuit(nbits, ngates, two_qubits_ratio = 0.2, t_ratio = 0.1)
```

Comment:
we should implement this directly as a normal `@device` function, this is over complicated.

Reply:
for validation, we will just directly evaluate things in YaoLang instead of constructing IR.

Reply:
besides, this implementation won't compile... you didn't construct the correct `IRCode`, let just remove this function in this PR for now, I'll do it in YaoCompiler.